app/views/events/_form.html.erb

```diff
@@ -28,11 +28,7 @@
 
   <div class="field">
     <%= f.label :occurs_at %> <small class="vermelho">*</small><br />
-    <% if @event.new_record? %>
-      <%= f.text_field :occurs_at, :class => 'datepicker' %>
-    <% else -%>
-      <%= f.text_field :occurs_at, :class => 'datepicker', :value => l(@event.occurs_at, :format => :picker) %>
-    <% end %>
+    <%= f.text_field :occurs_at, class: 'datepicker' %>
```

Comment:
Sure, seems like `:format => picker` was a fix for a an old problem and won't be necessary anymore since there a validation of presence, right?